ledger-core/v2/network/pulsenetwork/distributor.go

```diff
@@ -67,13 +70,22 @@ func NewDistributor(conf configuration.PulseDistributor) (insolar.PulseDistribut
 }
 
 func (d *distributor) Init(ctx context.Context) error {
-	handler := hostnetwork.NewStreamHandler(func(context.Context, *packet.ReceivedPacket) {}, d.responseHandler)
-
 	var err error
-	d.transport, err = d.Factory.CreateStreamTransport(handler)
+	d.transport, err = d.Factory.CreateDatagramTransport(nil)
```

Comment:
You will get nil pointer dereference, if udp transport will receive a packet. I think you should create empty handler

Reply:
hm, only if someone will try to use public address for transport service. Which supposed to be empty anyway, because pulsar should not handle inbound connections (until pulsar network happens)

Reply:
I'll create panicHandler with more verbose text